contracts/NodeDelegator.sol

```diff
@@ -218,6 +223,19 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
             if (address(eigenPod) != address(0)) {
                 eigenAssets += address(eigenPod).balance;
             }
+
+            // Get any ETH in the EigenDelayedWithdrawalRouter
+            IEigenDelayedWithdrawalRouter.DelayedWithdrawal[] memory delayedWithdrawals = IEigenDelayedWithdrawalRouter(
```

Comment:
This is so terribly documented on EigenLayer's Delayed Router... The function call is correct since it is only fetching the delayed withdrawals that[ have not yet been completed](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/pods/DelayedWithdrawalRouter.sol#L128). One could easily mistakenly call [userWithdrawals](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/pods/DelayedWithdrawalRouter.sol#L123) which returns a struct will all delayed withdrawals (completed and non completed)